instrumentation/instaawssdk/invoke_lambda.go

```diff
@@ -0,0 +1,47 @@
+package instaawssdk
+
+import (
+	"github.com/aws/aws-sdk-go/service/lambda"
+	otlog "github.com/opentracing/opentracing-go/log"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	instana "github.com/instana/go-sensor"
+	"github.com/opentracing/opentracing-go"
+)
+
```

Comment:
I wondering why linter did not complain about missing godoc here 🤔

Reply:
I ran linter once more, it shows no error. Anyway, I will add a godoc here.